### ISSUE TITLE
Rename project to "Open Science Carpentry"

### ISSUE THREAD

**Issue by konrad** [OKScienceDE/Open_Science_101#56]
Software/Data Carpentry was/is a strong inspiration for the format of the and the way the material is compiled. Other project also adapted the name an we would fit very well in this community. I suggest to send an email to the Software Carpentry mailing list and ask if they would see a problem with this.

**Comment by konrad** [comment]
A counter argument would be that the *Carpentry project are all focused on hand-on course ware while we are teaching abstract concepts.

**Comment by matthiasfromm** [comment]
@konrad Yes, the counter argument is a very strong one since "carpentry" refers specifically to a skill. Since this one is rather about the underlying concepts it might suggest a focus that we're not aiming for.

However, 101 might also be slightly misleading though the meaning refers to introduction/basics so it still includes what we're aiming for.

I'll also give it a thought.

**Comment by matthiasfromm** [comment]
Just playing around with a couple of terms:

* _Curriculum_ and syllabus are probably too narrow.
* _Compendium_ weakens a bit of "teaching resource provision" aspect, since it usually refers to the whole body of knowledge, does it?
* A _course_ or _module_ is too strictly related to the form of teaching.
* _101_ is mainly referring to introductions and basics, so it might be too short in the long term.
* _Repository_ might be too technical, though it opens up to the fact that people can contribute.
* _Digest_ is probably more used in other contexts.
* _Directory_ might be too short in terms of "there is more than just an outline"?
* _Compilation_ might only work in combination with a specifier like "teaching"?

Would you have any other idea?

**Comment by aleimba** [comment]
* *Training* might be too focused on the training aspect
* *Material* too focused on material

I still would like to wait for the answer from Software Carpentry, but as we discussed with @konrad, @matthiasfromm is probably right that Carpentry indicates a hands-on-course. But of course it would be great to be under the Carpentry umbrella.

**Comment by band** [comment]
How about "Open Science Praxis". The word "praxis" is used in disciplines that require practical skills (e.g., nursing). However it can also be associated with the phrase "practical action". And perhaps that is a useful way to describe the objectives of Open Science 101: provide an understanding of the notions, policies, and practices that support the development and ongoing maintenance of open science.

**Comment by aleimba** [comment]
hm, *Praxis* is not too bad. Thanks, for the suggestion @band!
